app.css

```diff
@@ -0,0 +1,97 @@
+/* table{width: 100%; border: 2px black;} */
```

Comment:
Maybe consider removing the styles that are no longer relevant, I'll let you decide which ones they are